brut.apktool/apktool-lib/src/main/java/brut/androlib/Androlib.java

```diff
@@ -172,15 +172,10 @@ public void recordUncompressedFiles(ExtFile apkFile, Collection<String> uncompre
                     unk.getCompressionLevel(file) == 0 &&
                     unk.getSize(file) != 0) {
 
-                    if (StringUtils.countMatches(file, ".") > 1) {
+                    ext = FilenameUtils.getExtension(file);
+                    if (ext.isEmpty() || (!ext.equals("png") && StringUtils.countMatches(file, ".") > 1)) {
```

Comment:
Don't know if I feel comfortable with this patch. This happens with plenty of other attributes outside of PNG. I need to research a proper fix which might be detecting two extensions and saving as such.

Reply:
Sure there might be more extensions to consider, but currently the .9.png extension is the most prominent and is the only extension that spams the apktool.yml file and prevents Windows recompile. I think it's a solid workaround until/if we have a more universal one.

Reply:
Probably right there, I'll do some double checks and merge this

Reply:
Sorry for the delay, getting back into dev. It seems `.9.png` images are not compressed. So we should just add support to check for more than one extension and thus store ".9.png" into the list.

Reply:
The problem is that there is virtually no way to distinguish an extra extension from just a part of the file name. A dot is a valid character in a resource name. It's just that <code>.9.png</code> should get the same treatment as <code>.png</code>.

Reply:
I see, maybe we will start with a predefined list to get around the valid character. So we just look from the end of the string for `9.png` and work from there.